controllers/construct/mongodbstatefulset.go

```diff
@@ -64,6 +64,17 @@ const (
 	automationAgentLogOptions = " -logFile ${AGENT_LOG_FILE} -maxLogFileDurationHrs ${AGENT_MAX_LOG_FILE_DURATION_HOURS} -logLevel ${AGENT_LOG_LEVEL}"
 
 	MongodbUserCommand = `current_uid=$(id -u)
+declare -r current_uid
+if ! grep -q "${current_uid}" /etc/passwd ; then
+sed -e "s/^mongodb:/builder:/" /etc/passwd > /tmp/passwd
+echo "mongodb:x:$(id -u):$(id -g):,,,:/:/bin/bash" >> /tmp/passwd
+export NSS_WRAPPER_PASSWD=/tmp/passwd
+export LD_PRELOAD=libnss_wrapper.so
+export NSS_WRAPPER_GROUP=/etc/group
+fi
+
+`
+	MongodbUserCommandWithAPIKeyExport = `current_uid=$(id -u)
 AGENT_API_KEY="$(cat /mongodb-automation/agent-api-key/agentApiKey)"
```

Comment:
we might do this also by swallowing stderr to /dev/null.

Reply:
Initially I implemented it that way: https://github.com/mongodb/mongodb-kubernetes-operator/pull/1479/commits/6526377527089b4d9f598761c22deb57f4d7047a but then @irajdeep had a great comment that he thinks it would be nice to make sure that we can see the error message in appdb in case monitoring does not work for any reason.